test/integration/targets/postgresql/tasks/main.yml

```diff
@@ -207,6 +198,23 @@
   debug:
     msg: "{{ postgres_version_resp.stdout }}"
 
+- set_fact:
+    encryption_values:
+    - 'yes'
+
+- set_fact:
+    encryption_values: '{{ encryption_values }} + ["no"]'
+  when: postgres_version_resp.stdout is version('10', '<=')
+
+- include: test_password.yml
```

Comment:
include_tasks instead of include please, include will be deprecated

Reply:
This is just a PR to get the tests working on newer Fedora hosts and devel has already been merged with the changes as is. The original code had `include` in it so I just kept the changes at a minimum.

Reply:
i understand, but it's a single keyword :)

Reply:
I'm just being lazy, would have to create a new commit to devel and then backport it again to the 3 stable branches. Might do it tomorrow morning if I have time.